packages/liveblocks-client/package.json

```diff
@@ -40,7 +40,7 @@
     "@liveblocks/jest-config": "*",
     "@types/ws": "^8.5.3",
     "dotenv": "^16.0.3",
-    "msw": "^0.39.1",
+    "msw": "^2.0.12",
```

Comment:
MSW doesn't seem to be used in `@liveblocks/client` package but I updated it here anyway. I leave it to you to decide whether to remove it or not.